hibernate-core/src/main/java/org/hibernate/jpa/event/internal/CallbackBuilderLegacyImpl.java

```diff
@@ -20,10 +20,8 @@
 import javax.persistence.PersistenceException;
 
 import org.hibernate.MappingException;
-import org.hibernate.annotations.common.reflection.ClassLoadingException;
-import org.hibernate.annotations.common.reflection.ReflectionManager;
-import org.hibernate.annotations.common.reflection.XClass;
-import org.hibernate.annotations.common.reflection.XMethod;
+import org.hibernate.annotations.common.reflection.*;
```

Comment:
btw, do we have style rule regarding the `*` usage, btw?

Reply:
It should be avoided.